### ISSUE TITLE
Swift version?

### ISSUE THREAD

**Issue by NirajCapermint** [zsy78191/XXXTabbarController#3]
Anyone converted this library into swift version ? I know I can use Obj C in swift but I need this complete code in swift. Any help will be appreciated. Thanks !!

**Comment by zsy78191** [comment]
Hello, I don't recommend that you continue to use this library, or use swift to mimic the implementation of this library.
There are two reasons:
1. The development time of this library is earlier than iPhone X, so it can't fit the new model very well.
2. The library's own linear animation method actually has errors. Although the effect is not visible, it is recommended to use the POP framework instead.
3. This tabbar does not do very well in dealing with the lifeVC of the child VC.
In summary, I recommend that you use swift to re-implement a tabbar controller with animation effects. The animation principle is also very simple. It only involves frame changes, background color changes, position changes, and transparency changes. good luck.

**Comment by zsy78191** [comment]
Forgot to say the point. . I haven't studied swift, sorry.
